modules/capability/CHANGELOG.md

```diff
@@ -31,7 +31,35 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Dependencies
 
+<<<<<<< HEAD
 * [\#4068](https://github.com/cosmos/ibc-go/pull/4068) Upgrade capability module to cosmos-sdk v0.51.0
+=======
+* [\#6193](https://github.com/cosmos/ibc-go/pull/6193) Bump Cosmos SDK to v0.50.6.
+* [\#6193](https://github.com/cosmos/ibc-go/pull/6193) Bump `cosmossdk.io/store` to v1.1.0.
```

Comment:
I think this changelog can be removed. And we can open a PR to the release branch to bump cosmos-sdk and cometbft deps to align with latest ibc-go releases.